src/components/icon/icon.test.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Icon from './icon';
+
+describe('Icon', () => {
+  it('has defaults', () => {
+    const tree = renderer.create((
+      <Icon name={'foobar'} />
+    )).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  describe('Size API', () => {
+    it('renders small, medium, and large sizes', () => {
+      const sizes = ['small', 'medium', 'large'];
+      sizes.forEach((size) => {
+        const tree = renderer.create((
+          <Icon name={'foobar'} size={size} />
+        )).toJSON();
+        expect(tree.props.className).toContain(`size-${size}`);
+      });
+    });
+  });
+
+  describe('focusable API', () => {
```

Comment:
What does the `focusable` attribute on the SVG element do?

I tried setting it and tabbing into it but I can't in our deployment.

Reply:
TLDR; Ima remove it from here. Backstory...

I think I see what happened, my guess is Huth referenced:

https://www.w3.org/TR/SVGTiny12/interact.html#FocusableAttribute

But SVGTiny12 is not in browsers and I believe SVG 2.0 which may add something similar never finalized yet (I think). I think even `tabIndex` is flaky and/or will require some more research.

Nice catch, and I think I'll just remove this. Note, it's in our bigmaven codebase as well, so we may just want to leave that since it probably doesn't mess anything up but maybe we do.

Reply:
I agree that removing it is best for now. I also looked into it -- findings @ https://github.com/mavenlink/design-system/pull/42#discussion_r307064724